pulser/sequence.py

```diff
@@ -1177,9 +1177,10 @@ def config_slm_mask(self, qubits: Set[QubitId]) -> None:
 
         # Find tentative initial and final time of SLM mask if possible
         for channel in self._channels:
-            if self._channels[channel].addressing == "Global":
-                # Cycle on slots in schedule until the first pulse is found
-                for slot in self._schedule[channel]:
+            if not self._channels[channel].addressing == "Global":
+                continue
+            # Cycle on slots in schedule until the first pulse is found
+            for slot in self._schedule[channel]:
                     if not isinstance(slot.type, Pulse):
```

Comment:
You had to change the indentation after this block too

Reply:
Do you mean that at l.1184 the indentation is one too much?

Reply:
Yes, for that line and the ones after it. Once the `for` was shifted to the left, the statements under it had to be too

Reply:
To be precise, for lines 1184-1193.